target/linux/mediatek/dts/mt7988a-arcadyan-mozart.dts

```diff
@@ -326,8 +326,10 @@
 
 	pcie@0,0 {
 		reg = <0x0000 0 0 0 0>;
+		#address-cells = <3>;
+		#size-cells = <2>;
 
-		mt7996@1,0 {
+		mt7996@0,0 {
```

Comment:
Are you sure about changing the referenced PCIe slot?

Reply:
Yes, you are right, and as the node address should match what is in `reg`, it should have been `0,0` from the beginning. Thank you for clarifying that.